### PR TITLE
Pull request to test codebuild trigger on TensorFlow script mode

### PR DESCRIPTION
**Pull request by yangaws**
*Issue #, if available:*

*Description of changes:*
Test codebuild trigger

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://y519hrazpe.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=3cef0a7a-dcec-4fbb-86c2-c712389490c3%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: buildspec.yml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+      - gpu_fw_binary=$(basename $GPU_FRAMEWORK_BINARY)
+      - wget -O $build_dir/$gpu_fw_binary $GPU_FRAMEWORK_BINARY
+
+      - GPU_TAG="$FRAMEWORK_VERSION-gpu-py$GPU_PY_VERSION-$build_id"
+
+      - cd $build_dir
+      - docker build -f $gpu_dockerfile --build-arg py_version=$GPU_PY_VERSION --build-arg framework_installable=$gpu_fw_binary -t $PREPROD_IMAGE:$GPU_TAG .
+      - cd ../../
+
+
```
Comment (laurenyu) [inline buildspec.yml]: don't need the second newline

File: buildspec.yml [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        fi
+
+    finally:
+      # shut down remote gpu instance
+      - cleanup-gpu-instances
+      - cleanup-key-pairs
+
+      # remove ecr image
+      - aws ecr batch-delete-image --repository-name $ECR_REPO --region $AWS_DEFAULT_REGION --image-ids imageTag=$CPU_TAG
+      - aws ecr batch-delete-image --repository-name $ECR_REPO --region $AWS_DEFAULT_REGION --image-ids imageTag=$GPU_TAG
```
Comment (laurenyu) [inline buildspec.yml]: add a newline to the end of the file

File: buildspec.yml (hunk 1 above)
Comment (yangaws) [reply buildspec.yml]: Updated

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://8teunz1x41.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=11abaab9-3d61-4993-8271-43f62ae5b79a%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://8teunz1x41.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=468b1c8a-b362-46d4-aacc-4c8fdb3f0ac8%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://8teunz1x41.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=fda0497e-e65e-4c62-9940-de0300687ef6%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://8teunz1x41.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=9bd8e3bf-0387-477b-9244-4da667c7c787%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://8teunz1x41.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=a53c79a6-102b-4c9c-a24a-357ec28d1d07%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://8teunz1x41.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=753b4bc5-afdf-4136-bd81-bb90e3e56061%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Pull request merged**